src/store/profile.js

```diff
@@ -54,31 +54,37 @@ const GET_PROFILE = gql`
         description
         ssbURI
       }
-      threads {
+    }
+  }
+`
+
+const GET_PROFILE_THREADS = gql`
+  query($id: ID, $cursor: String) {
+    getThreads(id: $id, limit: 10, cursor: $cursor) {
```

Comment:
it's pretty unclear what the `id` is meant to be. Note for future I try to name it like `feedId` or `profileId` as a check for the human passing things in that they are passing the right sorta thing in